packages/notifier/README.md

```diff
@@ -171,103 +285,23 @@ const {
-iteration values with the `subscription` using the `AsyncIterable` API. Since each published value
-will be sent to all subscribers, the SubscriptionKit should generally not be used with rapidly produced values.
-
-An iteration’s  *suffix subset* is defined by its *starting point* in the original iteration. 
-  * A starting  point may be a non-final value or a termination. 
-  * The suffix subset has exactly the original iteration’s members from its starting point to and
-     Including its termination (e.g. if the original is { 2 5 9 13 Fail } with Fail as the termination and 
-     a starting point at 9, the subset is { 9 13 Fail }).
-  * When a value becomes available on the original iteration, it *promptly* becomes available
-     on every suffix subset whose starting point is at or before that value (e.g. if the original is
-     { 2 5 9 13 Fail } and 9 becomes available, 9 promptly becomes available to any suffix 
-     subset with a starting point of 2, 5, or 9. It does not become available to any subset starting at
-     13 or Fail).
-
-The values published using the publication define the original iteration. Each subscription has a starting
-point in that iteration and provides access to a suffix subset of that iteration starting at that starting
-point. The initial subscription created by the `makeSubscriptionKit()` call provides the entire iteration.
-Each subscription is an `ForkableAsyncIterable` capable of producing any number of `ForkableAsyncIterator`s,
-each of which advances independently from the subscription's starting point.
-Each produced `ForkableAsyncIterator` is an `AsyncIterator`,
-with a `fork()` method that when called
-produces a new `ForkableAsyncIterator`
-whose starting point is the current position of its parent `ForkableAsyncIterator`.
-
-Carol's code is like Bob's except lower level, using the `ForkableAsyncIterable` interface directly.
-
-```js
-import { makePromiseKit } from '@agoric/promiseKit';
-
-const subscriptionIterator = subscription[Symbol.asyncIterator]();
-const { promise: afterA, resolve: afterAResolve } = makePromiseKit();
-
-const observer = harden({
-  updateState: val => {
-    if (val === 'a') {
-      afterAResolve(subscriptionIterator.fork());
-    }
-    console.log('non-final', val);
-  },
-  finish: completion => console.log('finished', completion),
-  fail: reason => console.log('failed', reason),
-});
-
-observeIterator(subscriptionIterator, observer);
-// eventually prints
-// non-final a
-// non-final b
-// finished done
-
-// afterA is a Promise<ForkableAsyncIterator> so we use observeIterator on it.
-observeIterator(afterA, observer);
-// eventually prints
-// non-final b
-// finished done
-```
-## Summary
+It is also possible to use `subscribeEach` for forward-lossless consumption of a `subscriber`
+or `subscription`, and `subscribeLatest` for lossy consumption of a `subscriber` or `notifier`.
```

Comment:
IMO, this should be the headline and not just a footnote.  I think we should deprecate the `getSharable*Internals()` and `observeIterat*` manipulation and just point to using these adaptors with `for-await-of`, which has much clearer semantics.

Reply:
Do you have something concrete in mind? The current structure from this PR looks like
> * PublishKit and Related Types
> * Distributed Asynchronous Iteration
> * Type Differences
>   * Lossiness
>     * NotifierKit
>     * SubscriptionKit
>     * PublishKit
>   * Use Cases
> * Example
> * Distributed Operation
> * Summary

`subscribeEach` and `subscribeLatest` are mentioned in **Type Differences** and **Example** and **Distributed Operation**.